Makefile

```diff
@@ -1,13 +1,20 @@
+GOGO_PROTOBUF := `go list -f "{{.Dir}}" -m github.com/gogo/protobuf`
+
+.PHONY: all go rust binlog c++
+
 all: go rust binlog c++
 
-go:
-	./generate-go.sh
+dependence:
+	go mod download
+
+go: dependence
+	GOGO_PROTOBUF=${GOGO_PROTOBUF} ./generate-go.sh
```

Comment:
Now that we have protobuf definitions in the repository, do we still need GOGO_PROTOBUF?